src/Components/Web.JS/src/GlobalExports.ts

```diff
@@ -46,6 +49,7 @@ interface IBlazor {
     domWrapper: typeof domFunctions;
     Virtualize: typeof Virtualize;
     PageTitle: typeof PageTitle;
+    dispatchEvent: typeof EventRegistry.prototype.dispatchEvent;
```

Comment:
Does this have to be part of the pubternal API? Can't the JS code dispatch events on its own without having to call via `Blazor._internal.dispatchEvent`?